src/wallet/walletdb.cpp

```diff
@@ -977,6 +983,31 @@ DBErrors CWalletDB::ZapSelectTx(CWallet *pwallet, vector <uint256> &vTxHashIn, v
     return DB_LOAD_OK;
 }
 
+DBErrors CWalletDB::ZapSigmaMints(CWallet *pwallet) {
```

Comment:
please rename function to more descriptive name.

Reply:
What do you suggest? It's just derived from `ZapWalletTx`..

Reply:
After discussion, I agree with add comment instead.

Reply:
After discussion, added description of function operation to the header file. (Keeping similar format to `ZapWalletTx` for consistency)